rpm_spec/core-dom0-linux.spec.in

```diff
@@ -192,6 +192,28 @@ if [ $1 -ge 2 ]; then
     fi
 fi
 
+# setup thinpool autoextend
+if [ -e /etc/lvm/lvm.conf ]; then
+    tpool_threshold_line_value=$(grep -n -Po '^[^#]thin_pool_autoextend_threshold[ \t]*=[ \t]*\K\d+' /etc/lvm/lvm.conf)
+    if [ -n "$tpool_threshold_line_value" ]; then
+        tpool_threshold_line="$(echo "$tpool_threshold_line_value" | cut -d':' -f1)"
+        tpool_threshold_value="$(echo "$tpool_threshold_line_value" | cut -d':' -f2)"
+
+        prev_line="$(sed -n "$((tpool_threshold_line-1))p" /etc/lvm/lvm.conf)"
```

Comment:
We should be certain to have `set -eo pipefail` here.  We might also be able to use awk instead of echo + cut.

Reply:
I think it's already the case in RPM script section? Also, we can use `awk` if it's to avoid pipe.

Reply:
I don’t believe it is, actually.